lib/awful/rules.lua

```diff
@@ -236,17 +242,67 @@ rules.extra_properties = {}
 --
 -- To add a new properties, just do:
 --
---    function awful.rules.high_priority_properties.my_new_property(c, value)
+--    function awful.rules.high_priority_properties.my_new_property(c, value, props)
 --        -- do something
 --    end
 --
+-- By default, the table has the following functions:
+--
+-- * tag
```

Comment:
I should stop reviewing commits on GitHub. I saw a previous commit where this part of the code was already present. Thus, I'm again looking at  them in the wrong order....